### PR TITLE
refactor: replace built-in with `fmod` in `math/base/assert/is-prime`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   Completes the TODO to replace inbuilt `fmod` with `stdlib_base_fmod`, in [`math/base/assert/is-prime`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/assert/is-prime).

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/assert/is-prime/src/main.c [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #include "stdlib/math/base/special/sqrt.h"
 #include "stdlib/math/base/special/floor.h"
 #include "stdlib/constants/float64/max_safe_integer.h"
+#include "stdlib/math/base/special/fmod.h"
 #include <stdint.h>
 #include <math.h>
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/assert/is-prime/src/main.c]: @gunjjoshi Can we now also remove this header file?

Comment (gunjjoshi) [reply lib/node_modules/@stdlib/math/base/assert/is-prime/src/main.c]: Ah, yes. Done!

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi!

**Pull request merged**